sorc/global_cycle.fd/read_write_data.f90

```diff
@@ -1063,20 +1072,25 @@ SUBROUTINE NETCDF_ERR( ERR, STRING )
  RETURN
  END SUBROUTINE NETCDF_ERR
 
- !> Read file from the GSI containing the foundation temperature
- !! increments and mask.
+ !> Read increment file from the GSI containing either the foundation 
+ !! temperature increments and mask, or the soil increments. 
  !!
  !! The data is in NetCDF and on a gaussian grid. The grid contains two
  !! extra rows for each pole. The interpolation from gaussian to
  !! native grid assumes no pole points, so these are removed.
  !!
  !! @param[in] GSI_FILE Path/name of the GSI file to be read.
+ !! @param[in] FILE_TYPE file-type to be read in, 'NST' or 'LND'.
```

Comment:
I have not worked with the GSI lately.  The NST and land increments are in separate files?  And they are on the same gaussian grid?  Because I see variables IDIM_GAUS and JDIM_GAUS are used for both.

Reply:
I don't know anything about the NST increments I'm afraid. The land increments are created by the GSI on the Gaussian grid, using a branch that Jeff Whitaker currently owns (he hasn't merged it back to develop yet, and it probably has a few minor edits before its ready). I checked that the grid info in the land increment file is the same as that generated within the code for the NST files, and it was.